gradle/libs.versions.toml

```diff
@@ -6,7 +6,7 @@ kotlinx-coroutines = "1.10.2"
 kotlinx-datetime = "0.6.2"
 kotlinx-io = "0.7.0"
 kotlinx-serialization = "1.8.1" # check with IJ
-ktor3 = "3.0.3"
+ktor3 = "3.1.3"
```

Comment:
is it required? we'd better be in sync with our internal clients

Reply:
@sproshev will it be possible to merge this before July 7th? WDYT?